joschahenningsen/TUM-Live
# Feature/unified player bar in splitview

### Motivation and Context
In split view we have currently two player bars, where the bigger player's player bar is active and the other one is disabled.
To enable the sound controls we currently mute the smaller player and unmute the bigger player with the player bar.
As the players are not perfectly synchronous, if the PresentationSlide-Player is bigger, the sound can be a little off to the camera player.

### Description
Only the camera players sound and controlbar is used, so the sound is not off. Also the playerbar is aligned over 
the whole video to give the same user expereience than in the old player.

### Steps for Testing
Prerequisites:
- 1 Student
- 1 Livestream

1. Log in
2. Navigate to a Livestream
3. Open Split Player
4. Check if all the controls and seeking is working and the controlbar is aligned correctly
5. Check if subtitles are displayed correctly

### Screenshots
![Bildschirm­foto 2023-01-20 um 14 51 19](https://user-images.githubusercontent.com/17506411/213714284-8d9bc044-1ca4-49c1-b1f4-0ae67451e9f3.png)
nshots
